Error while checking build/O2/fullCI for 3715b202069ef621e60aa6bc5b5aca60d9304ce3 at 2023-06-09 22:32:
```
## sw/BUILD/O2-full-system-test-latest/log
command /sw/slc8_x86-64/O2/11470-slc8_x86-64-local1/prodtests/full-system-test/dpl-workflow.sh had nonzero exit code 1
[ERROR] error parsing options of o2-strangeness-tracking-workflow: unrecognised option '--disable-root-output'


## sw/BUILD/o2checkcode-latest/log
--
========== List of errors found ==========
++ GRERR=0
++ grep -v clang-diagnostic-error error-log.txt
++ grep ' error:'
++ GRERR=1
++ [[ 1 == 0 ]]
++ mkdir -p /sw/INSTALLROOT/affa6ffe10a7eb0b4ba4e32c6cc5d55b95665940/slc8_x86-64/o2checkcode/1.0-local579/etc/modulefiles
++ cat
--
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/11470/3715b202069ef621e60aa6bc5b5aca60d9304ce3/build_O2_fullCI/pretty.html).

--- a/Detectors/GlobalTrackingWorkflow/src/strangeness-tracking-workflow.cxx
+++ b/Detectors/GlobalTrackingWorkflow/src/strangeness-tracking-workflow.cxx
@@ -42,6 +42,7 @@ void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
   // option allowing to set parameters
   std::vector<o2::framework::ConfigParamSpec> options{
     {"disable-root-input", o2::framework::VariantType::Bool, false, {"disable root-files input reader"}},
+    {"disable-root-output", o2::framework::VariantType::Bool, false, {"disable root-files output writer"}},
     {"disable-mc", o2::framework::VariantType::Bool, false, {"disable MC"}},
     {"configKeyValues", VariantType::String, "", {"Semicolon separated key=value strings"}}};
   o2::raw::HBFUtilsInitializer::addConfigOption(options);
@@ -58,6 +59,8 @@ WorkflowSpec defineDataProcessing(ConfigContext const& configcontext)
   // Update the (declared) parameters if changed from the command line
   auto useMC = !configcontext.options().get<bool>("disable-mc");
   auto useRootInput = !configcontext.options().get<bool>("disable-root-input");
+  auto disableRootOut = configcontext.options().get<bool>("disable-root-output");
+
 
   o2::conf::ConfigurableParam::updateFromString(configcontext.options().get<std::string>("configKeyValues"));
   o2::conf::ConfigurableParam::writeINI("o2strangeness_tracking_workflow_configuration.ini");
@@ -68,7 +71,10 @@ WorkflowSpec defineDataProcessing(ConfigContext const& configcontext)
   o2::globaltracking::InputHelper::addInputSpecs(configcontext, specs, itsSource, itsSource, itsSource, useMC, itsSource);
   o2::globaltracking::InputHelper::addInputSpecsPVertex(configcontext, specs, useMC); // P-vertex is always needed
   o2::globaltracking::InputHelper::addInputSpecsSVertex(configcontext, specs);        // S-vertex is always needed
-  specs.emplace_back(getStrangenessTrackingWriterSpec(useMC));
+
+  if (!disableRootOut) {
+    specs.emplace_back(getStrangenessTrackingWriterSpec(useMC));
+  }
 
   // configure dpl timer to inject correct firstTFOrbit: start from the 1st orbit of TF containing 1st sampled orbit
   o2::raw::HBFUtilsInitializer hbfIni(configcontext, specs);

--- a/Detectors/GlobalTrackingWorkflow/src/strangeness-tracking-workflow.cxx
+++ b/Detectors/GlobalTrackingWorkflow/src/strangeness-tracking-workflow.cxx
@@ -61,7 +61,6 @@ WorkflowSpec defineDataProcessing(ConfigContext const& configcontext)
   auto useRootInput = !configcontext.options().get<bool>("disable-root-input");
   auto disableRootOut = configcontext.options().get<bool>("disable-root-output");
 
-
   o2::conf::ConfigurableParam::updateFromString(configcontext.options().get<std::string>("configKeyValues"));
   o2::conf::ConfigurableParam::writeINI("o2strangeness_tracking_workflow_configuration.ini");
   GID::mask_t itsSource = GID::getSourceMask(GID::ITS); // ITS tracks and clusters

--- a/prodtests/sim_challenge.sh
+++ b/prodtests/sim_challenge.sh
@@ -251,6 +251,11 @@ if [ "$doreco" == "1" ]; then
   taskwrapper svfinder.log o2-secondary-vertexing-workflow $gloOpt
   echo "Return status of secondary vertexing: $?"
 
+  echo "Running strangeness tracking flow"
+  #needs results of S.Vertexer + ITS reco
+  taskwrapper sttracking.log o2-strangeness-tracking-workflow $gloOpt
+  echo "Return status of strangeness tracking: $?"
+
   echo "Producing AOD"
   taskwrapper aod.log o2-aod-producer-workflow $gloOpt --aod-writer-keep dangling --aod-writer-resfile "AO2D" --aod-writer-resmode UPDATE --aod-timeframe-id 1 --run-number 300000
   echo "Return status of AOD production: $?"

